include/greeter/greeter.h

```diff
@@ -26,6 +26,9 @@ namespace greeter {
      * @return a string containing the greeting
      */
     std::string greet(LanguageCode lang = LanguageCode::EN) const;
+
+    /// @brief Return an iso date string
+    std::string getIsoDate() const;
```

Comment:
While I do like the idea of adding an example library in the main project, let's try to keep the example simple. How about just changing the `std::string greet()` method to use the `fmt` library for string concatenation instead?

Reply:
This is also possible. 

My version is an minor, but compatible interface extensions i.e. `v1.1.0` -> `v1.2.0`

Reply:
Sure, however the example library is not intended to be consumed by users. It should only show a simple example library implementation using best-practices. Therefore I feel that the library itself should be consistent and minimal as I would expect from others, even if the use-case is made-up. This new method does not really fit into a "say-hello" library.

Reply:
This project is intended as a template, so it should show how different use cases can be handled with `cmake`
and `CPM-cmake`

If I plan to use it for one component of a bigger project, like `boost`, to build and distribute a lot of libraries.

How useful should it be?

How can I use it to develop each lib standalone, but some libs depends on an other.

Like `boost::filesystem`, which depends on `boost::system`.

Some are header only, linke boost::asio.

Other depends on `boost::asio` like `boost::beast`